### ISSUE TITLE
Declaration with External accessProfile fails

### ISSUE THREAD

**Issue by gwolfis** [F5Networks/f5-appsvcs-extension#246]
### Environment
 * Application Services Version: 3.17.1
 * BIG-IP Version: 13.1.1

### Summary
Declaring an accessProfile which is published external (eg. Github) will not get deployed successfully through AS3. Downloading the policy and import it into a BIG-IP works both for tar and tar.gz files.

The used accesprofile simply requests a login before showing the pool member content (user/user).

### Steps To Reproduce
1. Submit the following declaration:
```json
 {
    "class": "AS3",
    "action": "deploy",
    "declaration": {
        "class": "ADC",
        "schemaVersion": "3.15.0",
        "id": "Deploying_App_Services",
        "label": "Deploying_App_Services",
        "remark": "Deploying_App_Services",
        "App_Services": {
            "class": "Tenant",
            "HTTPS_Service": {
                "class": "Application",
                "template": "https",
                "serviceMain": {
                    "class": "Service_HTTPS",
                    "virtualAddresses": [
                        "10.1.10.121"
                    ],
                    "snat": "auto",                    
                    "pool": "web-pool",
					"profileHTTP": "basic",
					"serverTLS": "webtls",
					"profileAccess": {
						"use": "accessProfile"
					}
                },
                "web-pool": {
                    "class": "Pool",
                    "monitors": [
                        "http"
                    ],
                    "members": [
                      {
                            "servicePort": 80,
                            "serverAddresses": [
                              "10.1.20.11",
							  "10.1.20.12"
                            ]
                        }
                    ]
                },
				"webtls": {
                    "class": "TLS_Server",
                    "certificates": [
                        {
                            "certificate": "webcert"
                        }
                    ]
                },
                "webcert": {
                    "class": "Certificate",
                    "certificate": {
                        "bigip": "/Common/default.crt"
                    },
                    "privateKey": {
                        "bigip": "/Common/default.key"
                    }
                },
                "accessProfile": {
                	"class": "Access_Profile",
                	"url": "https://raw.githubusercontent.com/gwolfis/forum2020-f5-automation/master/profile_Common_test-access-policy.conf.tar.gz",
                	"ignoreChanges": true
                }
            }            
        }
    }
 }

### Expected Behavior
The 
### Actual Behavior
Results in a 500 error with the message of declaration failed.

{
    "results": [
        {
            "code": 500,
            "message": "declaration failed",
            "response":"�\b

**Comment by jmcalalang** [comment]
I haven't tested this theory, but I think you need to have your schema at 3.16..

https://clouddocs.f5.com/products/extensions/f5-appsvcs-extension/latest/declarations/access-related.html?highlight=accessprofile#referencing-an-external-iam-policy-using-a-url

**Comment by gwolfis** [comment]
I have updated the schema version to 3.16 but this still results in the same behavior.

**Comment by focrensh** [comment]
Just to confirm, i was able to reproduce with just a TAR and not GZ as the docs mentioned.

**Comment by focrensh** [comment]
Tracking with AT-1436

**Comment by gwolfis** [comment]
I have tested with BIG-IP 15.0.1 in AWS and AS3 3.18. Created the access policy on the BIG-IP. Tested it. Unzipped from tar.gz to .tar. Exported it into S3 and removed the access policy from the BIG-IP and declarared it via AS3 external URI. This works!
Uploaded the same access policy in Github and failed.

**Comment by dstokesf5** [comment]
This has been resolved as part of the 3.21.0 release.
